stream-analytics-queries/production-job-delayed/production-job-delayed.asaql

```diff
@@ -87,14 +86,13 @@ SELECT
   jobId AS jobId,
   sensorId AS sensorId,
   producedQuantity AS actualQty,
-  CASE
-    WHEN expectedQuantity > requestedQuantity THEN requestedQuantity
-    ELSE expectedQuantity
-  END AS expectedQty,
+  expectedQuantity AS expectedQty,
```

Comment:
So it is fine to show some number in the notifications that can be greater than the requested quantity?

Reply:
Well, it would be wrong to show a notification in which we are not able to explain why the notification is generated.
And we can't really not show the notification for the cases when an order breaches the promised time.

What would be ideal will be to show a percentage of delay in time but that would also need changes in F&O.
I will log a bug to fix in future along with feedback from customers.